### ISSUE TITLE
A11y_OSvisualupdate_MiniControGallery_ CalenderandColorPickerControls_Keyboard : The keyboard focus is not clearly visible for the color picker control.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/microsoft-ui-xaml#1584]
GitHubTags: #A11y_OSvisualupdate, #A11yMAS, #A11yTCS, #Win32, #DesktopApp, #WindowsUILibrary, #MAS2.4.7, #Keyboard

**Environment Details:**
Application Name: OS visual update
Windows Version: Windows10
Screen Reader: Narrator, NVDA 2019.2

**Steps to Reproduce:**
1. Launch Mini Control Gallery App from the installer location.
2. Screen with Calender and Color Picker controls would appear on right side of the screen.
3. Navigate to "Calender and Color Picker Controls" section.
4. Using "Tab" key navigate to the color picker control and observe if the keybaord focus is clearly visible or not.

**Actual:**
When keyboard users navigate to the color picker control the keyboard focus is not clearly visible due to which the users are unable to define where exactly the keyboard focus is on the screen.

**Expected:**
The keyboard focus should be clearly visible for the Color Picker Control while navigation.

**User Impact:**
If the keyboard focus is not clearly visible for the controls, then the keyboard only users will have difficulty in understanding where their keyboard focus has moved while navigation. 

**Attachment**
![A11y_ControlUIupdatebasedonSantoriniOSvisualupdate_MiniControGallery_ CalenderandColorPickerControls_Keyboard  The keyboard focus is not clearly visible for the color picker control](https://user-images.githubusercontent.com/39204909/68756733-ada9ba80-0630-11ea-82ca-99b62c6d0b87.gif)

**Comment by jevansaks** [comment]
We've had an internal issue tracking this for a while, but we can use this one going forward. Problem is that the focus visualization in the color area doesn't have great contrast so it is a little hard to tell where focus is. On the other hand, the tooltip is visible when you're doing this so maybe that's enough of a clue?

This is what was spec'd as the focus solution so we would need design to recommend a different solution to address this.

**Comment by YuliKl** [comment]
The tooltip is acting as a supplemental focus indicator. I believe that in practice users won't have any trouble finding the currently-focused element, so closing this issue.
